factcast-store/src/main/java/org/factcast/store/registry/validation/FactValidationAspect.java

```diff
@@ -46,10 +47,9 @@ public Object interceptPublish(ProceedingJoinPoint joinPoint) throws Throwable {
   }
 
   private void validate(List<? extends Fact> facts) {
-
```

Comment:
dunno but maybe we should use a dedicated thread pool here instead of the common one?
Just to ensure that "validation" load is an isolated thing

Reply:
done